src/SFA.DAS.EmployerAccounts.Jobs/RunOnceJobs/SeedAccountUsersJob.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Authorization;
+using SFA.DAS.EmployerAccounts.Data;
+using SFA.DAS.EmployerAccounts.ReadStore.Data;
+using SFA.DAS.EmployerAccounts.ReadStore.Models;
+using SFA.DAS.EmployerAccounts.Types.Models;
+
+namespace SFA.DAS.EmployerAccounts.Jobs.RunOnceJobs
+{
+    public class SeedAccountUsersJob
+    {
+        private readonly IRunOnceService _runOnceService;
+        private readonly IAccountUsersRepository _accountUsersRepository;
+        private readonly Lazy<EmployerAccountsDbContext> _db;
+        private readonly ILogger _logger;
+        private readonly string _jobName;
+
+        public SeedAccountUsersJob(IRunOnceService runOnceService, IAccountUsersRepository accountUsersRepository, Lazy<EmployerAccountsDbContext> db, ILogger logger)
+        {
+            _runOnceService = runOnceService;
+            _accountUsersRepository = accountUsersRepository;
+            _db = db;
+            _logger = logger;
+            _jobName = typeof(SeedAccountUsersJob).Name;
+        }
+
+        [NoAutomaticTrigger]
+        public Task Run()
+        {
+            return _runOnceService.RunOnce(_jobName, MigrateUsers);
+        }
+
+        public async Task MigrateUsers()
+        {
+            var users = _db.Value.Memberships.Include("User").Where(x=>x.Role != Role.None).AsEnumerable().ToList();
```

Comment:
Could the strongly typed `Include` overload be used instead? e.g. `.Include(m => m.User)`. I think it's in the `System.Data.Entity` namespace.